src/services/webhookServices.ts

```diff
@@ -1,2 +1,54 @@
-//placeholder for webhook services
-//to trigger CI pipeline
+import logger from "../utils/logger";
+import { WebhookPayload, DispatchPayload } from "../models/webhookModels";
+import { EVENT_TYPE } from "../utils/constants";
+import { getChannel } from "../config/rabbitmq";
+import { rabbitMQConfig } from "../config/rabbitmq";
+
+export const createWebhookService = () => {
+  const publishToRabbitMQ = async (payload: DispatchPayload): Promise<void> => {
+    try {
+      const channel = getChannel();
+      channel.publish(
```

Comment:
```suggestion
      const channel = getChannel();
      if (!channel) {
        throw new Error('Failed to get a valid RabbitMQ channel');
      }
      channel.publish(
```

Just some defensive code to make sure that we do get a channel before publishing otherwise we will have an unhandled error.